solutions/src/main/java/ru/sberbank/school/task07/AnyStringNumberImpl.java

```diff
@@ -0,0 +1,28 @@
+package ru.sberbank.school.task07;
+
+import lombok.NonNull;
+
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AnyStringNumberImpl implements AnyStringNumber<List<String>> {
+
+    private FileParser parser;
+
+    public AnyStringNumberImpl(FileParser parser) {
+        this.parser = parser;
+    }
+
+    @Override
+    public List<String> findStringsByNumbers(@NonNull String pathToFile, @NonNull Integer... numbers) throws FileNotFoundException {
```

Comment:
Line is longer than 120 characters (found 132).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a777325279>*